src/main/java/analyzer/AnnotatorAnalyzer.java

```diff
@@ -19,30 +18,35 @@
     private static final Pattern PATTERN_AUTHOR_NAME_FORMAT = Pattern.compile(REGEX_AUTHOR_NAME_FORMAT);
     private static final int MATCHER_GROUP_AUTHOR_NAME = 1;
 
-
+    /**
+     *
+     */
```

Comment:
I'd rather not have empty header comments. We can hold back the rule to require header comments for all public members if it is too much work to enforce retroactively.